### DISCUSSION TITLE
日文日期之歌

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#964] [General]
網址：https://www.youtube.com/watch?v=ZJSy4fqc3a4
有人跟我一樣修了日文結果日期一直背不起來嗎？
我最近發現了一個日文日期之歌，還蠻好聽的，
太多聽幾遍後有好背一點。

**Comment by Evan0979** [comment]
最近正好想學個日文但不知從何學起,感謝分享

**Comment by ghost** [comment]
透過旋律來增強記憶點，對於背下很有幫助，但每當你想要知道後方日期，假設31日，你就要從1唱到31，總之有利有弊吧

**Reply by ru0372572** [reply]
好好笑，跟注音或英文字母一樣，每次都要唱很久

**Comment by swerfg03joe** [comment]
學習外文最簡單的方法就是常常練習，與其找一些方法硬是把他背下來，不如多看外文作品，不管是報章雜誌或世上國外論壇，簡單的日常詞語會一直出現，漸漸的就記起來了，同時你還會知道在甚麼樣的情況使用他。

**Comment by Tuoresa** [comment]
透過歌曲學習真的是無論哪種語言，甚至是背誦知識感覺都有很大幫助欸
調用弦律久了也就形成長期記憶了，非常實用
感謝同學分享！！

**Comment by Sylvie0601** [comment]
感謝分享!一直很想好好學習日文，歌曲真的會幫助學習許多，就像學習英文時常聽英文歌、看美劇、聽英文廣播這種都是很棒的輔助，我也是自從培養看美劇、常聽英文歌、一直接觸歐美文化，我的英文就進步很多，甚至還可以學習更道地、口語的英文!我相信學習日文也可以藉由這些幫助讓人們更熟悉日文的。

**Comment by Kotori0501** [comment]
感謝分享
學習日文真的不是只有背單字文法這麼簡單而已，通常這樣的學習方式真的很難記起來
所以我通常都看日本vtuber直播，等直播後在看看一些網路的幫忙翻譯成中文的影片藉此學習，然後看動漫也是可以，可以先試著關字幕，遇到不會的就可以開字幕了，查看不懂的單字詞。

**Comment by s96015welson** [comment]
之前上日文課時老師也播放過，不過我還是常常忘記，我認為要熟悉語言我認為還是要回歸到練習和日常使用上。我自己喜歡看日本綜藝節目，遇到沒有字幕的時候就當作在練習日文

**Comment by delia07152** [comment]
好有趣，好像不論是從以前到現在只要關於需要被起來的東西，都會編出一段歌曲，而且有些莫名的好聽的？！

**Comment by ru0372572** [comment]
除了用歌曲記東西的，念書的時候也有很多怪口訣，像是「生生-反生」、「反燃-生燃」，「俄德法美日奧義英」或是「4cos(x)^3-3cos(x)」之類的，真的是忘不了
